src/app/client/src/app/modules/learn/components/course-consumption/course-consumption-header/course-consumption-header.component.html

```diff
@@ -65,7 +65,7 @@ <h5 class="textbook__title sb-color-primary font-weight-bold mb-0">{{courseHiera
               </button>
 
                <!-- Discussion Forum -->
-               <app-access-discussion *ngIf="isConnected" [fetchForumIdReq]="fetchForumIdReq" (routerData)="assignForumData($event)"> </app-access-discussion>
+               <app-access-discussion *ngIf="isConnected && fetchForumIdReq" [fetchForumIdReq]="fetchForumIdReq" (routerData)="assignForumData($event)"> </app-access-discussion>
```

Comment:
https://github.com/Sunbird-Ed/SunbirdEd-portal/commit/89a69ebf9f6e539ee85925f0e6584ec09da16f44#diff-dad6dc19ab527fd9d9090124531d57afb93f05cfd19a34083dd78c9de7a72338

If condition was checking for forms.length before.

Reply:
It was before when we are fetching all the thing from the component itself, now it is moved to "app-access-discussion" component